.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*.cs]
+end_of_line = lf
+indent_style = tab
+tab_width = 4
```

Comment:
I absolutely want this but don't want the "Use \n consistently" commit. Can you do a new PR with just this?